services/gcp/cloudsql/databases.yml

```diff
@@ -0,0 +1,31 @@
+name: Google Cloud SQL
+description: >-
+  The Cloud SQL database is the set of software and files that operates the MySQL, PostgreSQL, or SQL Server database service.
+scope: CRITICAL
+notes: >-
+  Cloud SQL is used to store and serve sensitive and application-critical data. Breach of a cloud SQL database can lead to exfiltration of highly sensitive data, or interruption of mission-critical applications.
+privileges:
+  create:
+    risks: [impact:spend, impact:consumption]
+    notes: >-
+      This privilege enables users to create databases within a Cloud SQL instance. It involves the potential excessive costs due to resource usage.
+  delete:
+    risks: [destruction:data, destruction:infra, destruction:logs]
+    notes: >-
+      With this privilege, users can delete databases within a Cloud SQL instance. It poses risks to data integrity and potential impact on the infrastructure.
+  get:
+    risks: [discovery:infra]
+    notes: >-
+      Access to database metadata.
+  list:
+    risks: [discovery:infra]
+    notes: >-
+      Access to database metadata.
+  update:
+    risks: []
```

Comment:
@nbrahms What risks do you think this poses? I'm not sure what the impact of changing collation settings would be-- I tried looking into this and it seems like it could affect certain queries (sorting/comaprisions and case sensitivity) but I'm not sure what kind of risk that is

Reply:
I guess there could be some sort of denial-of-service, if queries make specific assumptions about these settings.

But, in general, probably no real impact.

Reply:
Makes sense, I'll keep it as no risks.